guides/release/testing/unit-testing-basics.md

```diff
@@ -233,8 +233,8 @@ module('Unit | Service | employees', function(hooks) {
     const someThing = this.owner.lookup('service:some-thing');
 
     class MockPerson {
-      firstName = 'John';
-      lastName = 'Smith';
+      title = 'Dr.';
+      name = 'Zoey';
```

Comment:
Line 243 needs to be updated to say `Dr. Zoey is now an employee`.

Reply:
As an aside (this should be addressed in a separate PR), I don't think the assertion in line 244 would work at all. `someThing.employees[0]` and `person` are both class instances. We should probably remove this line.

Reply:
Addressed in 39a216a8

Will open a separate PR for line 244's assertion.

Reply:
PR opened:  #1535